tests/Shared/WorkloadTesting/ToolCommand.cs

```diff
@@ -59,6 +57,15 @@ public ToolCommand WithEnvironmentVariables(IDictionary<string, string>? extraEn
         return this;
     }
 
+    public ToolCommand WithEnvironmentVariableRemoved(string key)
```

Comment:
Is this method called?

Reply:
Oops, this is from the workload tests PR.

Reply:
Should I drop it for this PR?

Reply:
> Should I drop it for this PR?

Yes, I would only add it when it is called.